cooperative_lanechange/src/cooperative_lanechange.cpp

```diff
@@ -589,17 +613,20 @@ namespace cooperative_lanechange
         std::vector<double> times;
         trajectory_utils::conversions::speed_to_time(downtracks, final_actual_speeds, &times);
 
+        // Remove extra points
+        ROS_DEBUG_STREAM("Before removing extra buffer points, future_geom_points.size()" << future_geom_points.size());
+        int end_dist_pt_index = basic_autonomy::waypoint_generation::get_nearest_index_by_downtrack(future_geom_points, wm_, ending_state_before_buffer_);
+        future_geom_points.resize(end_dist_pt_index);
```

Comment:
Should the resizing here and in the next two lines be to index + 1?

Reply:
that's a good catch. Changed to + 1